src/Components/PresentationalComponents/Snippets/PageLoading.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Bullseye } from '@patternfly/react-core';
+import { Spinner } from '@patternfly/react-core';
+
+export default () => (
```

Comment:
This is not good practice as React will have to figure out some display name for the component. The shared fec `aslant` configure is set to warn about this, unfortunately `vulnerability-ui` is not (yet) using the shared config.